pages/about/groups/task-forces/pdf-aam.md

```diff
@@ -44,49 +44,55 @@ The [PDF-AAM Task Force Work Statement](/about/groups/task-forces/pdf-aam/work-s
 
-- Teleconferences of the Working Group and its task forces (also see [meeting minutes](https://www.w3.org/WAI/ARIA/task-forces/pdf-aam/minutes));
-- Face to face meetings;
-- [Email lists](#mailing-lists);
-- IRC discussion on the [#pdf-aam](irc://irc.w3.org/pdf-aam) IRC channel, used largely for minute-taking;
-- [PDF-AAM source repository issue tracker](https://github.com/w3c/pdf-aam/issues).
-- [ARIA specifications repository](https://github.com/w3c/aria);
-- [Web-Based Surveys (WBS)](https://www.w3.org/2002/09/wbs/159734/);
+- Teleconferences of the ARIA Working Group and its task forces (also see [meeting minutes](https://www.w3.org/WAI/ARIA/task-forces/pdf-aam/minutes))
+- Face-to-face meetings
+- [Mailing lists](#mailing-lists)
+- IRC (Internet Relay Chat) discussion on the [#pdf-aam](irc://irc.w3.org/pdf-aam) IRC channel, used largely for minute-taking
+- [PDF-AAM source repository issue tracker](https://github.com/w3c/pdf-aam/issues)
+- [ARIA specifications repository](https://github.com/w3c/aria)
+- [Web-Based Surveys (WBS)](https://www.w3.org/2002/09/wbs/159734/)
 
-These tools are used by participants of the Task Force. For ways non-participants can contribute, see [how to contribute to the Working Group and file comments](/about/groups/ariawg/contribute/).
+These tools are used by the Task Force participants. For ways that non-participants can contribute to the ARIA Working Group, including filing comments on publications, see [ARIA Contribution](/about/groups/ariawg/contribute/).
 
 ### Teleconferences
 
-The PDF-AAM Task Force meets the third Wednesday of the month, 1:00 - 1:30 PM Eastern Time. For more details, see [PDF-AAM Calendar](https://www.w3.org/groups/tf/pdf-aam/calendar).
+The PDF-AAM Task Force meets the third Wednesday of the month, 1:00–1:30 PM Eastern Time. For more details, including the meeting time in local time zones, see [PDF-AAM Calendar](https://www.w3.org/groups/tf/pdf-aam/calendar).
 
 ### Meeting Minutes
 
 [Minutes from previous meetings](https://www.w3.org/WAI/ARIA/task-forces/pdf-aam/minutes) are available.
 
 ### Mailing Lists
 
-The PDF-AAM Task Force uses the public-pdf-aam@w3.org mailing list ([mailing list archives](http://lists.w3.org/Archives/Public/public-pdf-aam/)) for email discussion. Participants are automatically added to the mailing list when they become a participant of the Task Force.
+The PDF-AAM Task Force uses the following mailing list for email discussion: [public-pdf-aam@w3.org Mail Archives](http://lists.w3.org/Archives/Public/public-pdf-aam/). 
+
+Participants are automatically added to the mailing list when they become a participant of the Task Force.
 
 ## Current Work
 
-The [Editors Draft](https://w3c.github.io/pdf-aam/) is expected to contain current PDF Accessibility API mappings.
+For the current PDF Accessibility API mappings, see the [Editor’s Draft](https://w3c.github.io/pdf-aam/).
```

Comment:
@tamsinewing555 Thanks for this suggestion. There is still no mappings in the Editors Draft. This is why I said "It is expected to contain". They've just put a couple of examples for now.

Reply:
Ah, that makes sense! I've lost track of your original sentence, can you give me the full sentence again or show me how to find it?